components/Ui/Link/UiLink.vue

```diff
@@ -1,60 +1,83 @@
 <template>
-  <component
-    :is="isNuxtLink ? 'NuxtLink' : 'a'"
+  <NuxtLink
     v-track-click="segment"
-    :href="isAnchor && url"
-    :to="isNuxtLink && url"
-    :style="hasLink && 'cursor:pointer'"
-    :rel="isExternal && 'noopener'"
-    :target="isExternal ? '_blank' : undefined"
+    :external="computedExternal"
+    :target="computedTarget"
     :title="title"
+    :to="url"
     @click="emit('click')"
     @mouseenter="$emit('mouseenter')"
   >
     <slot />
-  </component>
+  </NuxtLink>
 </template>
 
 <script setup lang="ts">
 import { CtaClickedEventProp } from "~/types/segment";
 
+type NuxtLinkTarget =
+  | "_blank"
+  | "_parent"
```

Comment:
That's true, we don't have those scenarios. They are also not the most common in _regular_ websites, but I wanted to include them to give NuxtLink the whole support, just in case.